newsrevenuehub/rev-engine
# remove stripe account from contributions api

#### What's this PR do?
- Removes `org_stripe_id` from response object of `/api/v1/contributions`.
- Removes the usage of `org_stripe_id` from frontend where it's being used, which is not necessary. https://github.com/newsrevenuehub/rev-engine/blob/c4caabee6b158be95a1d3c3fe076bad856eeb558/spa/src/components/contributor/contributorDashboard/EditRecurringPaymentModal.js#L37

#### Why are we doing this? How does it help us?
Endpoint `/api/v1/contributions` is exposing `org_stripe_id`, which is confidential information and can potentially be abused by bad actors.

#### How should this be manually tested?
Send a Get request to `/api/v1/contributions` and `org_stripe_id` shouldn't come in the response.
Update payment method should work as it is. (I was unable to test this end-to-end in my local)

#### Have automated unit tests been added?
No

#### How should this change be communicated to end users?
NA

#### Are there any smells or added technical debt to note?
NA

#### Has this been documented? If so, where?
NA

#### What are the relevant tickets?
**(Note: Please use syntax [JIRA-123] to link any relevant tickets)**
[DEV-1752]

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?
NA

#### Are there next steps? If so, what? Have tickets been opened for them?
NA
